include/boost/graph/karp_minimum_mean_cycle.hpp

```diff
@@ -0,0 +1,79 @@
+#ifndef BOOST_GRAPH_KARP_MINIMUM_MEAN_CYCLE_HPP
+#define BOOST_GRAPH_KARP_MINIMUM_MEAN_CYCLE_HPP
+
+#include <iostream>
+#include <boost/graph/graph_traits.hpp>
+#include <boost/graph/adjacency_list.hpp>
+
+namespace boost
+{
+    template<typename Graph>
+    double karp_minimum_mean_cycle(Graph g)
```

Comment:
Although it doesn't roll off the tongue very well, the paper does call it the "minimum cycle mean".